registries/_format/byte.md

```diff
@@ -1,9 +1,11 @@
 ---
 owner: DarrelMiller
 issue: 
-description: base64 encoded characters
+description: base64 encoded characters as defined in [RFC4648](https://www.rfc-editor.org/rfc/rfc4648#section-4)
 base_type: string
 layout: default
+oas_version: < 3.1
+remarks: "When using OpenAPI 3.1 or above it's recommended not to use this format and instead use [`contentEncoding` with a value of `base64`](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentencoding)."
```

Comment:
Streamlining:

> In OpenAPI 3.1, instead use [contentEncoding: base64url](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentencoding), optionally alongside [contentMediaType](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentmediatype)

I might also tack on:

> Note that [one example] in OAS 3.0.3 erroneously shows `format: base64` in place of `format: byte`

Reply:
Updating the former, not sure I understand the latter?

Reply:
@baywet search for `byte` and `base64` in the OAS 3.0.3 specification document and see if you can figure out what's going on :-)  I'm not trying to be annoying here, it really doesn't make sense.

@darrelmiller would it be reasonable to put out a 3.0.4 just to fix the `format: byte`/`format: base64` confusion?  I found a [2nd place](https://spec.openapis.org/oas/v3.0.3#special-considerations-for-multipart-content) that `base64` is used instead of `byte`

Reply:
There seems to be confusion in the spec between binary, byte and base64. It feels like byte and base64 are being used interchangeably and that sometimes binary has been used when byte should have been used instead.
This would go a long way explaining my initial confusion when implementing the type mapping in Kiota. And working through this PR has clarified some of it.
But again, maybe I'm reading things wrong.

Reply:
@baywet you are reading things correctly.  I've filed #3174 to address the `byte`/`base64` thing in OAS 3.0.4

I have not noticed `binary` being used where `byte` was intended - could you point any such places out for me if you remember them?  I can file an issue for that as well.

Reply:
Thanks for filing this. It'll help drive this PR home. May I suggest we leave things as is for those formats and revisit once things have been clarified on the spec side?

Disregard the comment about binary, it was in a multipart form data payload and I had missed that aspect.

Reply:
If by "leave this as-is" you mean ignore my "I might also tack on" bit about the current error in 3.0.3, yes, you can ignore that for this PR.  If you meant something else, please clarify.  Thanks for your own clarification about binary!

Reply:
yes that's what I meant, I already implemented the streamlining comment. resolving.